core/src/main/java/nu/yona/server/subscriptions/entities/BuddyConnectResponseMessage.java

```diff
@@ -15,11 +14,6 @@
 @Entity
 public class BuddyConnectResponseMessage extends BuddyConnectMessage
 {
-
-	@Transient
-	private UUID destinationID;
```

Comment:
The change of moving the backward relationship up from ``BuddyConnectResponseMessage`` to ``Message`` is good.